### PR TITLE
Updated: enhancement for New customer order vendor notification - Add link in the order number redirecting to /pro-dashboard/orders page #729

### LINKED ISSUE #729: enhancement for New customer order vendor notification - Add link in the order number redirecting to /pro-dashboard/orders page
**Is your feature request related to a problem? Please describe.**
The issue is related with the ticket: https://secure.helpscout.net/conversation/1090206726/7186
 As of now , there's no link in the notification for the vendors for new orders, the client would like to have a link that would send the vendors to their pro-dashboard/orders page so they could easily manage their orders after checking the email.

**Describe the solution you'd like**
Image+2021-01-20+at+2 35 57+PM

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by ketulsheth85**
### All Submissions:

* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

Closes #729 .

### How to test the changes in this Pull Request:

1. Check the pro plugin is enabled.
2. Put an order with the vendor product.
2. As admin user mark that order as processing.
3. Check vendor email order number has the link to the pro dashboard.

### CODE REVIEW

**Review by digitalchild — COMMENTED** [review]
Could we add an action here instead of that we could hook into within pro? thus not having to need to do a class detect here?

File: templates/emails/vendor-order-details.php [hunk 1]
```diff
@@ -23,6 +23,15 @@
 	<?php
 	$before = '';
 	$after  = '';
+	
+	$order_before_after = apply_filters( 'wcvendors_order_number_link_in_email_template', $before, $after );
```
Comment (digitalchild) [inline templates/emails/vendor-order-details.php:27]: Emails use actions for hooking into various aspects not a filter. If we had a simple do_action here we could move all of that logic into the pro code.

**Review by digitalchild — CHANGES_REQUESTED** [review]
Add an action here instead of a filter.

File: templates/emails/vendor-order-details.php (hunk 1 above)
Comment (ketulsheth85) [reply templates/emails/vendor-order-details.php:27]: If we move full Order Id to the pro by do_action then those who don't have the pro plugin that time in their email template this Order Id will be not appended.

**Review by digitalchild — APPROVED** [review]

**Pull request merged**